extensions/amp-story/1.0/amp-story.js

```diff
@@ -2264,6 +2266,21 @@ export class AmpStory extends AMP.BaseElement {
       });
   }
 
+  /**
+   * Loads amp-story-education if the viewer capability is provided.
+   * @private
+   */
+  maybeLoadStoryEducation_() {
+    if (!this.viewer_.hasCapability('education')) {
```

Comment:
Sorry, I think I'm not actually sure what this buys us after all.  There might be some experiences that we think old viewers should get.

If the viewer doesn't support education, it won't respond to the education viewer messages.  So I feel like we'd need this only if we want to differentiate between viewers that support education but don't respond vs viewers that don't support education.  But we don't need to distinguish that, right?

Reply:
It mostly allows lazy loading actually, so we don't have to load the extension all the time.

> is this just for the onboarding screens? i.e. if we were to implement another set of screens that don't show up on the initial load, we would need another separate extension?

This PR doesn't show any screen at all actually. It loads the extension, that will then, in a followup PR, start sending viewer messages to know if it should display an education screen.
We'll be in control of when to send these messages, on initial load or anything else (the extension has access to the store service).